docs/user-guide/howto.rst

```diff
@@ -327,7 +327,8 @@ Note that you can create your own style with matplotlib (see
 `here <https://matplotlib.org/stable/tutorials/introductory/customizing.html>`__ and
 `here <https://matplotlib.org/stable/tutorials/colors/colormaps.html>`__)
 
-The CTA observatory released a document describing best practices for **data visualisation in a way friendly to
+TODO: update the link
```

Comment:
Can you make this a comment?

Reply:
sure! The link has disappear on the CTAO web site, reason I put the TODO. I have asked to Alba to bring back this document